blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxReader.java

```diff
@@ -559,4 +612,91 @@ public Boolean apply(Row<Locator, Long> row) {
         return excessEnumMetrics;
     }
 
+    private <T extends Rollup> Points<T> getEnumStringValuesFromHashes(Points<T> points, ColumnList<Long> enumvalues) {
```

Comment:
Let's made these and the two other function below public.  They're usable for other stories.